Booking availability documentation (#57)
cultuurnet/uit-documentatie: content/json-ld-crud-api/latest/events/event-bookingavailability.md

---
---

# Event - bookingAvailability

> The bookingAvailability indicates whether there are still tickets or seats available for an event The following values for 'bookingAvailability' are applicable:

Reviewer: ```suggestion
The bookingAvailability indicates whether there are still tickets or seats available for an event. The following values for 'bookingAvailability' are applicable:
```

* `available`: there are still tickets or seats left for the event, or the capacity for the event is unlimited.

> * `unavailable`: the event is sold out or fully booked.

Reviewer: Should be `Available` and `Unavailable` (capitalized). Might also need to be fixed in other places.

Author: Thanks 👍 . Fixed it

It is possible to set the bookingAvailability on top event level or on subEvent level:
* **top event level**: for the whole event and all its subEvents (occurences). All subEvents will inherit the bookingAvailability from the top event level

> * **subEvent level**: for a one or multiple subEvent(s). The bookingAvailability on top event level will automtically be derived from the (different) bookingAvailability values on subEvent level.

Reviewer: ```suggestion
* **subEvent level**: for one or multiple subEvent(s). The bookingAvailability on top event level will automatically be derived from the (different) bookingAvailability values on subEvent level.
```

## Scope

It is possible to set a bookingAvailability for the calendarTypes `single`or `multiple`. 

> It is **not** possible to set a bookingAvailability for events with calendarType `permanent` or `periodic`. 

Reviewer: ```suggestion
It is **not** possible to change the bookingAvailability for events with calendarType `permanent` or `periodic`. They always have a `bookingAvailability` that is set to `Available`.
```
Just a suggestion.
